internal/host/connectivitycheckconvertor.go

```diff
@@ -46,9 +46,15 @@ func convertInterfacesToConnectivityCheckHost(hostId *strfmt.UUID, interfaces []
 		var ipAddresses []string
 		connectivityNic.Mac = hostInterface.MacAddress
 		connectivityNic.Name = hostInterface.Name
+
 		for _, ip := range hostInterface.IPV4Addresses {
 			ipAddresses = append(ipAddresses, strings.Split(ip, "/")[0])
 		}
+
+		for _, ip := range hostInterface.IPV6Addresses {
+			ipAddresses = append(ipAddresses, strings.Split(ip, "/")[0])
+		}
+
```

Comment:
We should also add IPv6 addresses to the test data here: https://github.com/openshift/assisted-service/blob/master/internal/host/connectivitycheckconvertor_test.go#L41 since they are being appended in the code above.